common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -1139,6 +1139,11 @@ public final class LC {
     public static final KnownKey smtp_to_lmtp_enabled = KnownKey.newKey(false);
     public static final KnownKey smtp_to_lmtp_port = KnownKey.newKey(7024);
 
+    @Supported
```

Comment:
what does this @Supported mean?

Reply:
This annotation represents a supported local config setting. To make a new setting show up in the zmlocalconfig -i command, use this annotation